### DISCUSSION TITLE
Clear Linux on Orange Pi5

### DISCUSSION THREAD

**Discussion by lah64** [Joshua-Riek/ubuntu-rockchip#263] [Ideas]
Dear Joshua,

I heart that Clear LInux is very fast... more than Ubutnu: https://community.clearlinux.org/t/benchmark-clear-linux-vs-the-rest/7565 

Could you please to prepare images for SoC + Clear Linux?
I think it's not bad idea or make instruction how prepare it.

**Comment by joshuataylor** [comment]
It's an Intel product, focused on amd64 only.

You can get similar results to clearlinux by using their kernel patches (like cachyos, xanmod, etc uses) and custom compiling packages.

Also phoronix doesn't (or didn't, I think it changed lately) use custom power profiles (eg cpupower set as performance), where as clearlinux uses this OOTB.
